src/backend/wire/protocol.cpp

```diff
@@ -0,0 +1,671 @@
+//
+// Created by Siddharth Santurkar on 31/3/16.
+//
+
```

Comment:
Instead of this, I suggest to put peloton header

```
//===----------------------------------------------------------------------===//
//
//                         Peloton
//
// expr_transformer.cpp
//
// Identification: src/backend/wire/protocol.cpp
//
// Copyright (c) 2015-16, Carnegie Mellon University Database Group
//
//===----------------------------------------------------------------------===//
```